src/geff/metadata_schema.py

```diff
@@ -123,6 +124,7 @@ class GeffMetadata(BaseModel):
     )
     directed: bool
     axes: Sequence[Axis] | None = None
+    affine: Affine | None = None
```

Comment:
Let's add validation that the dimensionality of the affine transform matches the number of spatial axes